iree/compiler/Dialect/Flow/Transforms/Passes.td

```diff
@@ -119,4 +119,10 @@ def StripAndSplatConstantVariables :
   let constructor = "mlir::iree_compiler::IREE::Flow::createStripAndSplatConstantVariablesPass()";
 }
 
+def InterchangeGeneric :
+    Pass<"iree-interchange-generic-op", "FuncOp"> {
```

Comment:
Nit: please make this `iree-flow-interchange-generic-op` (for now). Eventually this pass will move out of Flow.